packages/web-components/src/components/ic-search-bar/ic-search-bar.tsx

```diff
@@ -48,59 +48,64 @@ let inputIds = 0;
 })
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export class SearchBar {
-  @Element() el: HTMLIcSearchBarElement;
-
-  private inputId = `ic-search-bar-input-${inputIds++}`;
-  private inputEl: HTMLIcTextFieldElement;
-
-  private menuId = `${this.inputId}-menu`;
-  private searchSubmitButton: HTMLIcButtonElement;
-  private menu: HTMLIcMenuElement;
-  private menuCloseFromMenuChangeEvent: boolean = false;
-
   private anchorEl: HTMLElement;
-  private truncateValue = false;
-
   private assistiveHintEl: HTMLSpanElement = null;
-  private preventSubmit: boolean = false;
   private debounceAriaLive: number;
-
-  private timeoutTimer: number;
-  private prevNoOption: boolean = false;
   private hasTimedOut = false;
+  private inputEl: HTMLIcTextFieldElement;
+  private menu: HTMLIcMenuElement;
+  private menuCloseFromMenuChangeEvent: boolean = false;
   private preLoad = true;
-  private retryViaKeyPress: boolean;
+  private preventSubmit: boolean = false;
+  private prevNoOption: boolean = false;
   private retryButtonClick: boolean = false;
+  private retryViaKeyPress: boolean;
+  private timeoutTimer: number;
+  private truncateValue = false;
+  private searchSubmitButton: HTMLIcButtonElement;
+
+  private inputId = `ic-search-bar-input-${inputIds++}`;
```

Comment:
Do these need moving so they're in alphabetical order with the other props

Reply:
I was having problems with e2e tests involving id's so I think I isolated and forgot to move back!